data-explorer/kusto/api/powershell/powershell.md

```diff
@@ -35,13 +36,12 @@ For more information, see the [Azure Data Explorer client libraries](../netfx/ab
 #  ------------
 #  Packages location - This is an example of the location from where you extract the Microsoft.Azure.Kusto.Tools package.
 #  Please make sure you load the types from a local directory and not from a remote share.
-$packagesRoot = "C:\Microsoft.Azure.Kusto.Tools\Tools"
+$packagesRoot = "C:\Microsoft.Azure.Kusto.Tools\tools\net<ReplaceWithRightVersion>"
```

Comment:
>net<ReplaceWithRightVersion> [](http://example.com/codeflow?start=54&length=28)

Better make this a concrete example (e.g. `net472`), and add a comment what to do for other versions. Otherwise I'm sure there are people who are going to try <ReplaceWithRightVersion> not replaced at all...

Reply:
It's pretty standard in docs to place a field the user will need to replace within angle brackets, so I think it's probably ok.